overviewer_core/src/composite.c

```diff
@@ -113,7 +113,8 @@ alpha_over_full(PyObject* dest, PyObject* src, PyObject* mask, float overall_alp
     /* source position */
     int32_t sx, sy;
     /* iteration variables */
-    uint32_t x, y, i;
+    int32_t x, y;
+	uint32_t i;
```

Comment:
this is a tab indent

Reply:
Whoops, ended up just running `clang-format` on the C code to catch this and anything like it.